frontend/src/components/transactions/ReceiptRow.tsx

```diff
@@ -87,8 +82,9 @@ export interface Props {
 const ReceiptRow: FC<Props> = ({ receipt, transactionHash }) => {
   const { t } = useTranslation();
   let statusInfo;
-  if ("SuccessValue" in (receipt.outcome.status as RpcReceiptSuccessValue)) {
-    const { SuccessValue } = receipt.outcome.status as RpcReceiptSuccessValue;
+  if ("SuccessValue" in receipt.outcome.status) {
+    const { SuccessValue } = receipt.outcome.status;
+    // TODO: is this possible?
```

Comment:
@frol 
I need your advice :)

Reply:
If contract function returns void, then the success value will be `null`

Reply:
Is there an example of this?
This is `EXPERIMENTAL_tx_status` RPC call